fetch.sh

```diff
@@ -183,6 +185,24 @@ getwindowmanager () {
     fi
 }
 
+
+# Font (Configurable with "-F" and "--font" at launch)
+# Currently only supported for X Terminals
+if [ -f $HOME/.XResources ]; then
+    font=$(cat $HOME/.XResources | grep -v ! | awk '/font/ {print $2}' | sed 's/xft://;s/:pixelsize.*$//')
+elif [ -f $HOME/.Xresources ]; then
+    font=$(cat $HOME/.Xresources | grep -v ! | awk '/font/ {print $2}' | sed 's/xft://;s/:pixelsize.*$//')
+elif [ -f $HOME/.XDefaults ]; then
+    font=$(cat $HOME/.XDefaults | grep -v ! | awk '/font/ {print $2}' | sed 's/xft://;s/:pixelsize.*$//')
+elif [ -f $HOME/.Xdefaults ]; then
+    font=$(cat $HOME/.Xdefaults | grep -v ! | awk '/font/ {print $2}' | sed 's/xft://;s/:pixelsize.*$//')
+else
+    font="Unknown"
+fi
+
+# Screen Resolution (Configurable with "-R" and "--resolution" at launch)
+resolution=$(xdpyinfo | awk '/dimensions:/ {print $2}')
```

Comment:
This should be an if statement that also checks for xrandr as they can both be used to get screen resolution and the user may only have one installed.

You can check using "if type" like so:

    if type -p xrandr >/dev/null 2>&1; then